datadotworld/client/api.py

```diff
@@ -441,11 +525,290 @@ def download_datapackage(self, dataset_key, dest_dir):
 
         return path.join(dest_dir, 'datapackage.json')
 
+    # User Operations
+
+    def get_user_data(self):
+        """Retrieve data for authenticated user
+
+        Parameters
+        ----------
+        no parameters
+
+        Returns
+        -------
+        dict
+            User data, with all attributes
+
+        Raises
+        ------
+        RestApiException
+            If a server error occurs
+
+        Examples
+        --------
+        >>> import datadotworld as dw
+        >>> api_client = dw.api_client()
+        >>> user_data = api_client.get_user_data()
+        >>> user_data.displayName
+        'Name User'
+        """
+        try:
+            return self._user_api.get_user_data()
```

Comment:
This should return a dict instead of an object (i.e. `.to_dict()` should be called on what `self._user_api.get_user_data()` returns).